editor/project.clj

```diff
@@ -213,6 +213,7 @@
                                                     "-Ddefold.log.dir="
                                                     "-Djogl.debug.DebugGL" ; TraceGL is also useful
                                                     "-Djogl.texture.notexrect=true"
+                                                    "-XX:MaxRAMPercentage=75"
```

Comment:
This just affects the `:dev` profile, used for REPL builds. We need to up the limit to be able to build certain large projects.